README.md

```diff
@@ -38,6 +38,7 @@ See [Testing Farm onboarding guide](https://docs.testing-farm.io/Testing%20Farm/
 | `git_url` | An url to the repository with tmt metadata | url of repo that runs GHA |
 | `git_ref` | A tmt tests branch, tag or commit specifying the desired git revision which will be used for tests | `github.ref` when available, otherwise `master` |
 | `tmt_plan_regex` | A regular expression used to select tmt plans | all |
+| `tmt_plan_filter` | A filter used to select tmt plans | "" |
```

Comment:
I would also reference TF documentation of `plan_filter` for more information:

link: https://docs.testing-farm.io/Testing%20Farm/0.1/test-request.html#plan-filter

Reply:
Fixed by https://github.com/sclorg/testing-farm-as-github-action/pull/223/commits/9f61ef1cd42b030b982850456ca21d8ad11b6f75